src/stc/PlatWX.cpp

```diff
@@ -750,25 +750,25 @@ SurfaceFontDataD2D::SurfaceFontDataD2D(const FontParameters& fp)
         wxString faceName = fp.faceName;
         wxString extentTest(EXTENT_TEST);
         DWRITE_FONT_STYLE fontStyle;
-        DWRITE_TEXT_METRICS textmetrics = {0,0,0,0,0,0,0,0,0};
+        DWRITE_TEXT_METRICS textmetrics = { 0,0,0,0,0,0,0,0,0 };
```

Comment:
Hmm, does your editor _add_ spaces automatically on save? Some editors strip them automatically, but this is the first time I see spaces being added in otherwise unmodified places...

Reply:
Visual Studio did some auto-formatting when I closed the bracket of the function.